content/docs/1.6.0/references/helm-values.md

```diff
@@ -5,175 +5,173 @@ weight: 5
+| longhornDriver.priorityClass | string | `nil` | PriorityClass for Longhorn Driver. |
+| longhornDriver.tolerations | list | `[]` | Toleration for Longhorn Driver on nodes allowed to run Longhorn components. |
 
 ### Longhorn UI Settings
 
-Longhorn system contains user deployed components (e.g, Longhorn manager, Longhorn driver, Longhorn UI) and system managed components (e.g, instance manager, engine image, CSI driver, etc.).
-These settings only apply to Longhorn UI component.
+Longhorn consists of user-deployed components (for example, Longhorn Manager, Longhorn Driver, and Longhorn UI) and system-managed components (for example, Instance Manager, CSI Driver, and engine images). The following settings only apply to Longhorn UI.
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| longhornUI.nodeSelector | object | `{}` | Select nodes to run Longhorn UI |
-| longhornUI.priorityClass | string | `nil` | Priority class count for longhorn ui |
-| longhornUI.replicas | int | `2` | Replica count for longhorn ui |
-| longhornUI.tolerations | list | `[]` | Tolerate nodes to run Longhorn UI |
+| longhornUI.nodeSelector | object | `{}` | Node selector for Longhorn UI. Specify the nodes allowed to run Longhorn UI. |
+| longhornUI.priorityClass | string | `nil` | PriorityClass for Longhorn UI. |
+| longhornUI.replicas | int | `2` | Replica count for Longhorn UI. |
+| longhornUI.tolerations | list | `[]` | Toleration for Longhorn UI on nodes allowed to run Longhorn components. |
 
 ### Ingress Settings
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| ingress.annotations | string | `nil` | Ingress annotations done as key:value pairs |
-| ingress.enabled | bool | `false` | Set to true to enable ingress record generation |
-| ingress.host | string | `"sslip.io"` | Layer 7 Load Balancer hostname |
-| ingress.ingressClassName | string | `nil` | Add ingressClassName to the Ingress Can replace the kubernetes.io/ingress.class annotation on v1.18+ |
-| ingress.path | string | `"/"` | If ingress is enabled you can set the default ingress path then you can access the UI by using the following full path {{host}}+{{path}} |
-| ingress.secrets | string | `nil` | If you're providing your own certificates, please use this to add the certificates as secrets |
-| ingress.secureBackends | bool | `false` | Enable this in order to enable that the backend service will be connected at port 443 |
-| ingress.tls | bool | `false` | Set this to true in order to enable TLS on the ingress record |
-| ingress.tlsSecret | string | `"longhorn.local-tls"` | If TLS is set to true, you must declare what secret will store the key/certificate for TLS |
+| ingress.annotations | string | `nil` | Ingress annotations in the form of key-value pairs. |
+| ingress.enabled | bool | `false` | Setting that allows Longhorn to generate ingress records. |
```

Comment:
`Setting that allows Longhorn to generate ingress records for Longhorn UI service.`